tools/agentlint/internal/rivertags/rivertags.go

```diff
@@ -0,0 +1,248 @@
+// Package rivertags exposes an Analyzer which lints river tags.
+package rivertags
+
+import (
+	"fmt"
+	"go/ast"
+	"go/types"
+	"regexp"
+	"strings"
+
+	"golang.org/x/tools/go/analysis"
+)
+
+var Analyzer = &analysis.Analyzer{
+	Name: "rivertags",
+	Doc:  "perform validation checks on River tags",
+	Run:  run,
+}
+
+var noLintRegex = regexp.MustCompile(`//\s*nolint:rivertags`)
```

Comment:
In some places in our codebase we do something like this: `//nolint:golint,rivertags`. Can we amend the regex to catch this as well?

Reply:
Sure! Doing it at the regex level felt ugly, but I changed my approach to support this 👍